service/lxd.go

```diff
@@ -468,11 +468,18 @@ type CephDedicatedInterface struct {
 	Addresses []string
 }
 
+// OVNDedicatedInterface represents a dedicated interface for OVN.
+type OVNDedicatedInterface struct {
+	Network   api.Network
```

Comment:
Could this type get merged with CephDedidatedInterface? Seems the only difference is one includes the whole `api.Network` while the other just includes the `Type`.

Reply:
fixed